graphs/fit_atg_model.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+import pytest
+from scipy.optimize import least_squares
+
+# TODO(miskosz): Pass in as a parameter?
+EXPONENT = 6.23
+
+@dataclass
+class AtgModelFit:
+    """
+    Result of fitting (x, y) data samples to the curve:
+        y = (a/tg) * (x/tg)^6.23 * e^(-x/tg)
+    
+    TODO(miskosz): Maybe return also something like a standard deviation of the fit.
+    """
+    a: float
+    tg: float
+
+    def predict(self, x: float) -> float:
+        ys = _model(params=[self.a, self.tg], xs=[x])
+        return ys[0]
+
+
+def fit_atg_model(xs: np.ndarray, ys: np.ndarray) -> AtgModelFit:
```

Comment:
I think I have to learn Python but this is good style :+1:

Reply:
Pozerám, že Mišo je v Pythone na míle pred nami. Ja používam podobný zápis v C++, napríklad `auto generate_age(std::mt19937* generator) -> uint32_t`.